configure.ac

```diff
@@ -4791,6 +4791,13 @@ then
     ENABLED_DES3="yes"
 fi
 
+# DES3 ciphers
+AC_ARG_ENABLE([des3-ciphers],
```

Comment:
To avoid confusion, we need the new option to have "tls" in its name.  I suggest `--enable-des3-tls-suites`.